cookbooks/aws-parallelcluster-platform/resources/arm_pl/arm_pl_ubuntu22.rb

```diff
@@ -31,4 +26,8 @@ def armpl_platform
   def modulefile_dir
     "/usr/share/modules/modulefiles"
   end
+
+  def gcc_major_minor_version
+    '11.3'
```

Comment:
Why are we upgrading gcc as part of adding the support for Rocky/RHEL?

Reply:
According to the commit description:
> Different GCC versions are used for different operating systems. For existing operating systems, we keep the version unchanged (9.3 for alinux2, centos7, ubuntu20, rhel8, rocky8; 11.3 for ubuntu22). For newly added OSes (RHEL9 and Rocky9), we use 11.3 because 9.3 is not available.